apps/api/src/core/config/env.config.ts

```diff
@@ -7,7 +7,12 @@ const envSchema = z.object({
   // TODO: make required once billing is in prod
   POSTGRES_DB_URI: z.string().optional(),
   POSTGRES_MAX_CONNECTIONS: z.number({ coerce: true }).optional().default(20),
-  DRIZZLE_MIGRATIONS_FOLDER: z.string().optional().default("./drizzle")
+  DRIZZLE_MIGRATIONS_FOLDER: z.string().optional().default("./drizzle"),
+  DEPLOYMENT_ENV: z.string().optional().default("production"),
+  SENTRY_TRACES_RATE: z.number({ coerce: true }).optional().default(0.01),
+  SENTRY_ENABLED: z.enum(["true", "false"]).optional().default("false"),
+  SENTRY_SERVER_NAME: z.string().optional(),
+  SENTRY_DSN: z.string().optional()
```

Comment:
these var names need to be updated before next deploy, I'll add new names that before merging.